ocaml/stdlib/gc.mli

```diff
@@ -623,3 +623,20 @@ module Memprof :
        called on a profile which has not been stopped.
        *)
 end
+
+
+(** GC Tweaks are unstable and undocumented configurable GC parameters,
+    primarily intended for use by GC developers. *)
```

Comment:
Could you document the command-line syntax here?

Reply:
Good point, will do.